translations/EasyAdminBundle+intl-icu.da.xlf

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="da" datatype="plaintext" original="file.ext">
+    <header>
+      <tool tool-id="symfony" tool-name="Symfony"/>
+    </header>
+    <body>
+      <trans-unit id="4J45.H4" resname="login_page.username">
+        <source>login_page.username</source>
+        <target state="needs-l10n">__login_page.username</target>
```

Comment:
I guess you will be updating these as the last thing before merging?

Reply:
Missing translations added in https://github.com/itk-dev/dokk1gh/pull/5/commits/6ec6ed8c8b483ce2e9687c41f8a10daef574d8fd.